spec/system/jobseekers_can_subscribe_to_a_job_alert_spec.rb

```diff
@@ -135,8 +135,19 @@
     end
 
     context "when a location category search is carried out" do
+      let(:buffer_coordinates) { [5.004562496029994090, 56.50833566307333, 5.005710530794815389, 56.5051084208278] }
+      let(:buffer_hash) do
+        {
+          "5" => buffer_coordinates,
+          "10" => buffer_coordinates,
+          "15" => buffer_coordinates,
+          "20" => buffer_coordinates,
+          "25" => buffer_coordinates,
+        }
+      end
+
       before do
-        LocationPolygon.create(name: "london")
+        LocationPolygon.create(name: "london", buffers: buffer_hash)
```

Comment:
I think this would probably be better if we made a location polygon factory object